LICENSE-Paulov-t.md

```diff
@@ -0,0 +1,26 @@
+
+Any work or fork that is not Stay in Tarkov and uses any line of code from [Paulov-t](https://github.com/paulov-t), including but not limited to previous commits and forks of old commits is strictly prohibited. If you are using any [Paulov-t](https://github.com/paulov-t) code in your works, whether currently or previously, you can request permission directly from [Paulov-t](https://github.com/paulov-t) via GitHub, Email or Discord.
+
+**Redistribution**
+- You cannot reproduce and distribute copies of the Work (by [Paulov-t](https://github.com/paulov-t)) or Derivative Works (by [Paulov-t](https://github.com/paulov-t)) thereof in any medium, with or without modifications, and in Source or Object form.
+
+**Expiry of old NCSA Open Source License and this replacement**: 
+-  After the project was moved out of [Paulov-t](https://github.com/paulov-t) to [Stay In Tarkov](https://github.com/stayintarkov). [Paulov-t](https://github.com/paulov-t) was incorrectly added to the original [License](https://github.com/StayInTarkov/StayInTarkov.Client/commit/a2d111eeac96b4ddb15f5fc5a182330f25ddbe4f#diff-4673a3aba01813b595de187a7a6e9e63a3491d55821606fecd9f13a10c188a1d) with the message `Paulov-t - Original Stay In Tarkov (SIT) Code`. Original Stay in Tarkov (SIT) code means code merged from the [Paulov-t GitHub](https://github.com/paulov-t) and no code developed after the fact. You are therefore accepting LICENSE any original code written up to the point of the move to Stay In Tarkov organisation.
+- It added by Paulov that the current License is also **Unlicensed** [here](https://github.com/StayInTarkov/StayInTarkov.Client/commit/a2d111eeac96b4ddb15f5fc5a182330f25ddbe4f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) but also suggests that `None of my own work is Licensed. This is solely a just for fun project. I don't care what you do with it`. This is a poorly formatted phrase that does not grant a license. 
+- The README.md and therefore agreement was changed [here](https://github.com/StayInTarkov/StayInTarkov.Client/commit/638dea7394741496b617900a685a4f449f49b318). The LICENSE.md at that time did not and does not reflect any code written by [Paulov-t](https://github.com/paulov-t) for the Stay in Tarkov organisation. Key segment here is that LICENSE.md is not the only section that defined the License agreement. If you forked Stay in Tarkov on or after this commit, you are in fact accepting several LICENSE agreements that clashed and were not clear. All code by [Paulov-t](https://github.com/paulov-t) from this point is therefore UNLICENSED. See [GitHub Licensing](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license) for more information `without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work`
```

Comment:
Incorrect. Paulov-t's code is also under NCSA coverage as it was the only lawful license present.